products/cloudrun/api.yaml

```diff
@@ -272,7 +272,6 @@ objects:
     properties:
     - !ruby/object:Api::Type::NestedObject
       name: template
-      required: true
```

Comment:
Is this change intentional? Seems out of character with the rest of the changes.

Reply:
Yes!  I took that out because i saw its all flattened in terraform.yaml, and the flattened part is already required.